mlog/stdlog.go

```diff
@@ -85,3 +85,15 @@ func (l *loggerWriter) Write(p []byte) (int, error) {
 	}
 	return len(p), nil
 }
+
+type panicLoggerWriter struct {
+	logFunc func(msg string, fields ...Field)
+}
+
+func (l *panicLoggerWriter) Write(p []byte) (int, error) {
+	trimmed := string(bytes.TrimSpace(p))
+	if strings.HasPrefix(trimmed, "panic: ") {
```

Comment:
Being a stream this isn't guaranteed to be the total panic. Have you tried this in practice? Does it usually capture the panic?

Reply:
@iomodo, can we find a way to unit test this?

Reply:
I tested in practice, also wrote some unit tests. Given that `mlog` does not provide good testing capabilities I had to "hack it in". I'd suggest creating a separate ticket to add proper unit testing mechanism in `mlog`